webapp/src/constants.ts

```diff
@@ -52,7 +52,7 @@ export const CallAlertConfigs: { [key: string]: CallAlertConfig } = {
         type: CallAlertType.Warning,
         icon: 'alert-outline',
         bannerText: defineMessage({defaultMessage: 'Call quality may be degraded due to unstable network conditions.'}),
-        dismissable: false,
+        dismissable: true,
```

Comment:
I'd like to understand the effect of changing this because it was made not-dismissable by design given it's a condition affecting the client that could cause a degraded experience. Would the banner reappear at the next `mos` event that goes over the threshold or would it be permanently hidden?

Reply:
It will go away and then come back on the next `mos` event. I think we should think about the logic here and decide if we even want this banner to be dismissable.

Reply:
Either we implement a backoff or we keep it on.

Reply:
I would prefer not to add the complexity of a lock and a timer for this...

Reply:
I am on board :)

Reply:
Ok, I didn't want to use a lock and timer, I used a throttle. Here's the behavior: the call banner will appear instantly when it's first called, it will turn on and off based on mos just like always. But if you dismiss the banner, it won't come back at least for 20 seconds, then it will come back if there is still network trouble.  We can lengthen the delay if we want.

Reply:
And before you ask, I don't think we need to implement cross-popout/widget dismiss synchronization. 😛

Reply:
I was fine with just making it not dismissible so not going to ask for that :p